webapp/app/templates/basis/form_standard.html

```diff
@@ -6,7 +6,7 @@
     {% endblock %}
     {% for field in form %}
         <div class="form-row-center mt-3">
-            {{ macros.render_field(field) }}
+            {{ macros.render_field(field, first_field=field == render_info.form.first_field) }}
```

Comment:
For some reason render_info.form.first_field is not set for the StepReligion - thus, no focus^^

Reply:
The reason is that it overwrites the render function and thus it does not set it 🤦 In the future we could set this in the handle functions of the SteuerlotseStep and we would not have this problem :)

Reply:
We decided to keep it that way